Extensions/TileMap/helper/dts/tiled/Tiled.d.ts

```diff
@@ -0,0 +1,339 @@
+import { float, integer } from '../model/CommonTypes';
+/**
+ * Tiled JSON format.
+ */
+export declare type TiledMap = {
+  /** Hex-formatted color (#RRGGBB or #AARRGGBB) (optional) */
+  backgroundcolor?: string;
```

Comment:
all those properties are lowercase, should we prefer using camelCase? => `backgroundColor`?

Reply:
It's actually the definition of the JSON used by Tiled, a tile map editor.